file.go

```diff
@@ -136,19 +136,19 @@ func OpenFile(r io.ReaderAt, size int64, options ...FileOption) (*File, error) {
 // Only leaf columns have indexes, the returned indexes are arranged using the
 // following layout:
 //
-//	+ -------------- +
+//	---------------- +
```

Comment:
Go 1.19 adds some documentation formatting/rewrites. It wasn't happy with the initial layout here and kept trying to reformat it. It's happy with this format, if you don't mind it.

Reply:
Could we change the trailing ` +` into `--` as well for consistency?

Reply:
Done